vpdq/python/tests/test_util.py

```diff
@@ -25,3 +27,49 @@ def read_file_to_hash(input_hash_filename: str) -> t.List[vpdq.VpdqFeature]:
         hash.append(feature)
 
     return hash
+
+
+def generate_hashes(
```

Comment:
(ignorable, YMMV) This looks like a good place to force kwarg arguments by doing
```
def generate_hashes(
    *,
    output_folder: Union[Path, str],
```

This will prevent callers from doing 

```
generate_hashes("foo", ["bar"], true, 5, 10)
```


---

EDIT: Building off the question abou tgenerating the hashes as a CLI, the arguments of that CLI are probably just this, and then you do something like:

```
generate_vpdqs <list of files> --overwrite -d 250
```

Having it be a list of files then allow you to do shell expansion like dir/a/*.mp4

Reply:
Not looking to create a CLI, just trying to add another test.